src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write your code here
+const population =
+  [...document.querySelectorAll('.population')]
+    .map((el) => el.textContent)
+    .map((el) => el.split(',').join(''))
+    .reduce((el, acc) => parseInt(el) + parseInt(acc), 0) + '';
+
+const totalPopulation = population;
+
+const averagePopulation = (
+  Math.round(totalPopulation / population.length) + ''
+).split('');
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+total.textContent = totalPopulation
+  .split('')
+  .map((el, index, array) => {
+    if (array.length % 2 === 0) {
+      if (index === 1) {
+        return ',' + el;
+      }
+
+      if (index % 3 === 0) {
+        return el + ',';
+      }
+    } else {
+      if ((index + 1) % 3 === 0) {
+        return el + ',';
+      }
+    }
+
+    return el;
+  })
+  .join('')
+  .split(',')
+  .filter((el) => el !== '')
+  .join(',');
+
+average.textContent = averagePopulation
+  .map((el, index, array) => {
+    if (array.length % 2 === 0) {
+      if (index === 1) {
+        return ',' + el;
+      }
+
+      if (index % 3 === 0) {
+        return el + ',';
+      }
+    } else {
+      if ((index + 1) % 3 === 0) {
+        return el + ',';
+      }
+    }
+
+    return el;
+  })
+  .join('')
+  .split(',')
+  .filter((el) => el !== '')
+  .join(',');
```

Comment:
You can simplify your code like this
```suggestion
const people = [...document.getElementsByClassName('population')];
let total = 0;

people.forEach((population) => {
  total += +population.textContent.replace(/,/g, '');
});

const average = total / people.length;

document.querySelector('.average-population').innerText = average.toLocaleString('en-US');

document.querySelector('.total-population').innerText = total.toLocaleString('en-US');
```